src/pages/Pairing/Pairing.jsx

```diff
@@ -122,12 +151,13 @@ function Pairing() {
    * Fetch course timetable
    */
   useEffect(() => {
-    const fetchCourses = async () => {
-      const response = await getCourses();
-      setCourses(response);
-    };
+    // const fetchCourses = async () => {
+    //   // const response = await getCourses();
+    //    // await getCourses();
+    //   // setCourses(arr);
+    // };
 
-    fetchCourses();
+  // fetchCourses();
```

Comment:
Is this not an important code...? was this removed intentionally or was this supposed to be temporarily commented out as a debugging purposes? if it was supposed to be removed, then the entire useEffect code block should be removed and if it was only commented out for debugging purposes, this should be uncommented and reverted back to how it was before.